awx/ui/test/e2e/tests/test-users-crud.js

```diff
@@ -17,6 +17,16 @@ const store = {
     },
 };
 
+const searchUser = (client, users, username) => {
```

Comment:
We should consider adding this as a customCommand to the user page object if we suspect that this will be used many times.
http://nightwatchjs.org/guide/#writing-commands

Reply:
I've done that, thanks for the suggestion